src/features/api-keys/lib.ts

```diff
@@ -94,6 +94,25 @@ const $createApiKey = async (
 	return apiKey;
 };
 
+const getKeyMetadata = (reqBody: { name: any; description: any }) => {
+	const { name, description } = reqBody;
+
+	if (name != null && !(typeof name === 'string' || name instanceof String)) {
```

Comment:
I would expect this v to be enough, given how the deserialization works, which is how we do it in all other places as well
```suggestion
	if (name != null && typeof name !== 'string') {
```
That applies for description as well probably.

Reply:
We've gone through this before -- `string?` is *very* problematic because it actually has three states: `null/undefined`, `''` and some non-zero length string but for most intents and purposes the first two states mean the same thing: "i have no value". So I think ensuring empty string do not make it to the database is sensible.